src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -85,7 +85,8 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
 
         foreach ($lines as &$line) {
             $line = sprintf('%s%s', $prefix, $line);
-            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line));
+            //https://github.com/symfony/symfony/pull/14600
+            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line) + substr_count($line, '\<'));
```

Comment:
not only the line you mentioned is needed here but it's depend on the style that applied too.